php/EE/Runner.php

```diff
@@ -629,6 +636,12 @@ private function set_alias( $alias ) {
 
 	public function start() {
 
+		// // Protect 'cli info' from most of the runtime,
+		// // except when the command will be run over SSH
```

Comment:
Following phpcs wherever possible would be preferred.

Reply:
Done.